src/main/common/time.c

```diff
@@ -58,7 +69,7 @@ static rtcTime_t dateTimeToRtcTime(dateTime_t *dt)
 
 static void rtcTimeToDateTime(dateTime_t *dt, rtcTime_t *t)
```

Comment:
Pass rtcTime by value, ie `rctTime t`

Reply:
Should be cheaper to pass the pointer, since rtcTime_t is 64 bits. Not that it would make a significant difference.

Reply:
With ARM you get up to 4 parameters passed in registers very cheaply. Since `rtcTimeToDateTime` only has two parameters `rtcTime_t` will be passed in two registers. This may even be cheaper than passing by pointer.